src/HealthcareApis/HealthcareApis/help/Az.HealthcareApisService.md

```diff
@@ -0,0 +1,23 @@
+﻿---
+Module Name: Az.HealthcareApisService
+Module Guid: df9fc69a-c019-403a-9013-eefa7eb5e27a
+Download Help Link: https://docs.microsoft.com/en-us/powershell/module/az.healthcareapisservice
+Help Version: 0.1.0.0
+Locale: en-US
+---
+
+# Az.HealthcareApisService Module
```

Comment:
Regenerate the help after the changes above are made. This will automatically change to `# Az.HealthcareApis Module`.

Reply:
@poadhika this comment still applies